x/incentives/keeper/distribute.go

```diff
@@ -428,14 +428,9 @@ func (k Keeper) syncGroupWeights(ctx sdk.Context, group types.Group) error {
 	return nil
 }
 
-// syncVolumeSplitGroup syncs a group according to volume splitting policy.
-// It mutates the passed in object and sets the updated value in state.
-// If there is an error, the passed in object is not mutated.
-//
-// It returns an error if:
-// - the volume for any linked pool is zero or cannot be found
-// - the cumulative volume for any linked pool has decreased (should never happen)
-func (k Keeper) syncVolumeSplitGroup(ctx sdk.Context, group types.Group) error {
+// updatedGroupWeights calculates the updated weights of the group records based on the pool volumes.
+// It returns the updated group and an error if any. It does not mutate the passed in object.
+func (k Keeper) updatedGroupWeights(ctx sdk.Context, group types.Group) (types.Group, error) {
```

Comment:
```suggestion
func (k Keeper) calculateGroupWeights(ctx sdk.Context, group types.Group) (types.Group, error) {
```
<br/>
Maybe this would be more understandable?

Reply:
Agreed, changed here https://github.com/osmosis-labs/osmosis/pull/6651/commits/40db7972009bc0258f668dd0e6a34491ca5859dd